jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/DataTablesTests.java

```diff
@@ -264,7 +275,7 @@ public static void headerTableTest() {
     public static void itemTableTest() {
         itemTable.show();
         for (Chip chip : itemTable.getChips()) {
-            jdiAssert(itemTable.getColor(Integer.parseInt(chip.getText())), Matchers.is(chip.backgroundColor()));
+            chip.has().backgroundColor(itemTable.getColor(Integer.parseInt(chip.getText())));
         }
```

Comment:
Please, investigate opportunities to refactor method 'getColor'. It does not look reliable as it depends on concrete numbers.

Reply:
Resolved after a conversation